app/src/main/java/com/untitled/timespace/di/DatabaseModule.kt

```diff
@@ -0,0 +1,33 @@
+package com.untitled.timespace.di
+
+import android.content.Context
+import androidx.room.Room
+import com.untitled.timespace.data.local.TimeSpaceRoomDatabase
+import com.untitled.timespace.data.local.dao.CapsuleDao
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module(includes = [DatabaseModule.ProvideModule::class])
+internal class DatabaseModule {
+    companion object {
+        private const val DB_NAME = "time_space"
+    }
+
+    @Module
+    class ProvideModule {
+        @Provides
+        @Singleton
+        fun provideRoomDatabase(context: Context): TimeSpaceRoomDatabase {
+            return Room.databaseBuilder(context, TimeSpaceRoomDatabase::class.java, DB_NAME)
+                .fallbackToDestructiveMigration()
+                .build()
+        }
+
+        @Provides
+        @Singleton
+        fun provideCapsuleDao(roomDatabase: TimeSpaceRoomDatabase): CapsuleDao {
```

Comment:
```suggestion
        fun provideCapsuleDao(roomDatabase: TimeSpaceRoomDatabase): CapsuleDao = roomDatabase.capsuleDao()
```
이런 한줄만에 혹은 짧은 코드로 끝나는 함수 부분에선 저 같은 경우는 return을 넣지않고 이런 방식을 사용하는데 재환님은 이런 방식은 지양하나요?? 🤔